src/main/java/com/google/sps/servlets/CreateEventServlet.java

```diff
@@ -23,13 +23,13 @@
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
         KeyFactory keyFactory = datastore.newKeyFactory().setKind("event");
-        double price = request.getParameter("price");
-        long date = request.getParameter("date");
+        double price = Double.parseDouble(request.getParameter("price"));
+        long date = Long.parseLong(request.getParameter("date"));
```

Comment:
I trust you, but could you explain to me how this works, does it just parse the string returned from the form?